mava/specs.py

```diff
@@ -170,3 +170,11 @@ def get(self) -> Dict[str, Any]:
             _description_
         """
         return self.__dict__
+
+    def set(self, name: str, component: Any) -> None:
+        """_summary_
```

Comment:
Can we please add a descriptive dosctring here?

Reply:
Fixed :+1:

Reply:
Thank you :)